frontend/src/components/app/admin-routing.tsx

```diff
@@ -0,0 +1,14 @@
+import { adminRoutes } from './routes/routes';
+import { Route, Routes } from 'components/common/common';
+import { IRoute } from './common/interfaces';
+
+const AdminRouting: React.FC = () => (
+  <Routes>
+    {
+      adminRoutes.map(({ path, element }: IRoute) =>
```

Comment:
We need to add here AppRoutes. Because in this case if Admin `isAutorized === true`, he will not have login page